features/step_definitions/stream_steps.rb

```diff
@@ -14,6 +14,10 @@
   find(".toggle_nsfw_state", match: :first).click
 end
 
+When /^I toggle all nsfw posts$/ do
+  all("a.toggle_nsfw_state").each { |elt| elt.click }
```

Comment:
Space between { and | detected.<br>Pass `&:click` as an argument to `each` instead of a block.